examples/gno.land/p/demo/gnorkle/feed/task.gno

```diff
@@ -0,0 +1,7 @@
+package feed
+
+// Task is a unit of work that can be part of a `Feed` definition. Tasks
+// are executed by agents.
+type Task interface {
+	MarshalToJSON() ([]byte, error)
```

Comment:
btw -- this is inconsistent; you have `MarshalJSON` in other places (I suggest you stick to that)